Client/game_sa/CStreamingSA.cpp

```diff
@@ -257,7 +257,13 @@ void CStreamingSA::SetStreamingBufferSize(uint32 uiBlockSize)
     if (uiBlockSize == ms_streamingBufferSize * 2)
         return;
 
-    // Suspend old streaming handle
+    int pointer = *(int*)0x8E3FFC;
+    SGtaStream(&streaming)[5] = *(SGtaStream(*)[5])(pointer);
+
+    // Wait while streaming threads ends tasks
+    while (streaming[0].bInUse && streaming[1].bInUse)
```

Comment:
I think you've forgot a ;
```suggestion
    while (streaming[0].bInUse && streaming[1].bInUse);
```

Reply:
Unnecessary there